sysdata/sim/sim_data.py

```diff
@@ -104,6 +104,8 @@ def daily_prices(self, instrument_code: str) -> pd.Series:
 
         """
         instrprice = self.get_raw_price(instrument_code)
+        if len(instrprice) == 0:
+            raise Exception("No adjusted daily prices for %s" % instrument_code )
```

Comment:
misleading error message?

Reply:
Sorry, I thought sim_data handles only adjusted prices? And the resample-function crashed when I didn't have adjusted prices available for certain instrument